tests/validation/tests/v3_api/test_aks_cluster.py

```diff
@@ -53,26 +58,21 @@ def get_aks_version():
         print("JSON RESPONSE IS")
         print(response.content)
         json_response = json.loads(response.content)
-        versionarray_length = len(json_response)
-        aksclusterversion = json_response[versionarray_length-1]
-        print(aksclusterversion)
+
+        if multiple_versions:
+            aksclusterversion = [json_response[0], json_response[-1]]
```

Comment:
Is it possible there is only one version in the response? Should we handle that case?

Reply:
You're right, it probably won't ever happen but I think might be a possibility. I'll update all 3 of them to account for that!

Reply:
Fixed!